psi4/src/psi4/lib3index/df_helper.h

```diff
@@ -280,15 +283,24 @@ class DF_Helper {
     // => AO building machinery <=
     void prepare_AO();
     void prepare_AO_core();
-    void compute_AO_Q(const size_t start, const size_t stop, double* Mp,
+    void compute_dense_Qpq_blocking_Q(const size_t start, const size_t stop, double* Mp,
+                      std::vector<std::shared_ptr<TwoBodyAOInt>> eri);
+    void compute_sparse_pQq_blocking_Q(const size_t start, const size_t stop, double* Mp,
```

Comment:
+1 for the verbose function names here.  Makes it so much easier to understand at first glance